topgg/webhook.py

```diff
@@ -63,38 +66,38 @@ class WebhookManager(DataContainerMixin):
     A class for managing Top.gg webhooks.
     """
 
+    __slots__: t.Tuple[str, ...] = ("__app", "_webserver", "_is_running")
+
     __app: web.Application
     _webserver: web.TCPSite
-    _is_closed: bool
-    __slots__ = ("__app", "_webserver", "_is_running")
+    _is_running: bool
 
     def __init__(self) -> None:
         super().__init__()
-        self.__app = web.Application()
+
+        self.__app = None
```

Comment:
This line is causing errors in my existing code.
Reverting back fixes the issue

Reply:
Should be resolved.